micro_ros_agent/cmake/SuperBuild.cmake

```diff
@@ -19,14 +19,15 @@ unset(_deps)
 enable_language(C)
 enable_language(CXX)
 
-unset(xrceagent_DIR CACHE)
-find_package(xrceagent 2 EXACT QUIET)
+unset(microxrcedds_agent_DIR CACHE)
+find_package(microxrcedds_agent 2 EXACT QUIET)
 if(NOT xrceagent_FOUND)
-    ExternalProject_Add(xrceagent
+    ExternalProject_Add(microxrcedds_agent
             GIT_REPOSITORY
                 https://github.com/eProsima/Micro-XRCE-DDS-Agent.git
             GIT_TAG
-                v2.4.3
+                # TODO: Change when https://github.com/eProsima/Micro-XRCE-DDS-Agent/pull/375 is merged
+                feature/fastdds3
```

Comment:
Use a stable branch when ready and remove TODO

Reply:
As we need this for vulcanexus maybe we should have a new `vulcanexus-jazzy` branch and merge it there. For Vulcanexus we need a version in "x.x.x" format. Maybe we could take it from that branch. What do you think @pablogs9 ?

Reply:
IMO we shall not maintain a semversioned tag on a development branch that uses internally some other development branches. Maybe you can fork this repo for vulcanexus and apply patches there.